projects/igniteui-angular/src/lib/calendar/months-view/months-view.component.html

```diff
@@ -1,8 +1,14 @@
 <div class="igx-calendar__body">
-    <div class="igx-calendar__body-row--wrap">
-        <div [igxCalendarMonth]="month" [date]="date" [attr.tabindex]="activeMonth === month.getMonth() ? 0 : -1" (onMonthSelection)="selectMonth($event)" [index]="i" *ngFor="let month of months; index as i; trackBy: monthTracker">
+    <div  class="igx-calendar__body-row--wrap">
+        <span
+            [attr.aria-label]="month | date: 'LLLL'"
```

Comment:
It seems `aria-label` must be combined with something that supports one. Like this, on its own applied to a span, at lest NVDA for me reads months as "Clickable Dec" ignoring the nice "December" label :(